src/modules/movies.js

```diff
@@ -0,0 +1,77 @@
+import HelperFuncs from './helpers.js';
+
+class Movies {
+  constructor() {
+    this.API_BASE_URL = 'https://api.tvmaze.com/';
+    this.LIKES_URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/eeE9kmiK9ON5LxnmgZ0i/likes';
+    this.movies = [];
+    this.likes = [];
+  }
+
+  addLike = async (itemId, likeButton) => {
+    await fetch(this.LIKES_URL, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ item_id: itemId }),
+    }).then((response) => response.text(response)).then((json) => json);
+    await this.getLikes();
+
+    const index = this.likes.findIndex((like) => like.item_id === itemId);
+    const msgLikes = index >= 0 ? this.likes[index].likes : 0;
+    likeButton.nextElementSibling.innerHTML = msgLikes;
+  }
```

Comment:
- [ ] <b>OPTIONAL:</b> Currently one would be able to like a single movie more than once, which is not good because as you can understand, there wouldn't be any point of having to compare movies by likes if someone can give 1000+ likes to a single movie. It would be better to create a mechanism such that if you click on it for the second time, it removes the previous like instead of adding a new one.